File: 30-Days-of-Code/Day-2-Operators/Haskell/solution.hs
import System.IO
import Data.Char
import Text.Printf

main = do
    mealInput <- getLine
    tipInput <- getLine
    taxInput <- getLine
    
    let mealCost = read mealInput :: Float
    let tipPercent = read tipInput :: Int
    let taxPercent = read taxInput :: Int
    
    let tip = mealCost * (fromIntegral tipPercent / 100.0)
    let tax = mealCost * (fromIntegral taxPercent / 100.0)
    
    let totalCost = mealCost + tip + tax
    
    let roundedTotalCost = round totalCost
    
    printf "The total meal cost is %d dollars." (roundedTotalCost :: Int)

File: 30-Days-of-Code/Day-2-Operators/Python2/solution.py
mealCost = 0.0
tipPercent = 0
taxPercent = 0

mealCost = eval(raw_input())
tipPercent = eval(raw_input())
taxPercent = eval(raw_input())

tip = mealCost * (tipPercent / 100.0)
tax = mealCost * (taxPercent / 100.0)
totalCost = mealCost + tip + tax

roundedTotalCost = round(totalCost)

print 'The total meal cost is %d dollars.' % roundedTotalCost
